src/components/Nav.jsx

```diff
@@ -23,6 +23,22 @@ export default function Nav() {
 						</StyledNavLink>
 					</Link>
 				</StyledListItem>
+				<StyledListItem role="listitem">
```

Comment:
Why do you need to set the role to `listitem`? If you use an html `li` element, you shouldn't need to use it.